src/bundle/toolbar/utils.ts

```diff
@@ -0,0 +1,128 @@
+import {ToolbarName} from '../../modules/toolbars/constants';
+import {commonmark, defaultPreset, full, yfm, zero} from '../../modules/toolbars/presets';
+import {
+    EditorPreset,
+    ToolbarItem,
+    ToolbarsPresetOrEditorPreset,
+} from '../../modules/toolbars/types';
+import {
+    MToolbarData,
+    MToolbarItemData,
+    ToolbarDataType,
+    WToolbarData,
+    WToolbarItemData,
+} from '../../toolbar';
+import {MarkdownEditorViewProps} from '../MarkdownEditorView';
```

Comment:
plz add `type` modifier to the imports to indicate that only types are being imported